triples-generator/src/main/scala/ch/datascience/triplesgenerator/reprovisioning/OrphanMailtoNoneRemover.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2019 Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ch.datascience.triplesgenerator.reprovisioning
+
+import cats.effect.{ContextShift, IO, Timer}
+import ch.datascience.rdfstore.IORdfStoreClient.RdfDelete
+import ch.datascience.rdfstore.{IORdfStoreClient, RdfStoreConfig}
+import io.chrisdavenport.log4cats.Logger
+
+import scala.concurrent.ExecutionContext
+import scala.language.higherKinds
+
+private trait OrphanMailtoNoneRemover[Interpretation[_]] {
+  def removeOrphanMailtoNoneTriples(): Interpretation[Unit]
+}
+
+private class IOOrphanMailtoNoneRemover(
+    rdfStoreConfig:          RdfStoreConfig,
+    logger:                  Logger[IO]
+)(implicit executionContext: ExecutionContext, contextShift: ContextShift[IO], timer: Timer[IO])
+    extends IORdfStoreClient[RdfDelete](rdfStoreConfig, logger)
+    with OrphanMailtoNoneRemover[IO] {
+
+  override def removeOrphanMailtoNoneTriples(): IO[Unit] = queryWitNoResult {
+    s"""
+       |PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+       |PREFIX prov: <http://www.w3.org/ns/prov#>
+       |PREFIX dcterms: <http://purl.org/dc/terms/>
```

Comment:
the prefixes are not really being used - why not remove them?

Reply:
Sure, I've just removed them.